app/src/main/res/layout/activity_app_list_item.xml

```diff
@@ -68,7 +68,6 @@
             android:id="@+id/textViewDownloadProgress"
             android:layout_width="wrap_content"
             android:layout_height="wrap_content"
-            android:text="6.00 MB/12.00 MB   50%"
```

Comment:
Why was the sample text removed? Won't that make it more difficult to work on designing the XML view in Android Studio?

Reply:
Dump text should not be kept in our views because if there's a case the value is not set, it will show this text that does not make any sense to users.